honeycomb-core/src/cmapbuilder/grid/mod.rs

```diff
@@ -52,6 +52,17 @@ impl<T: CoordsFloat> CMapBuilder<T> {
 
 impl<T: CoordsFloat> CMapBuilder<T> {
     /// Create a [`CMapBuilder`] with a predefinite [`GridDescriptor`] value.
+    ///
+    /// # Arguments
+    ///
+    /// - `n_square: usize` -- Number of cells along each axis.
+    ///
+    /// # Generated Map
```

Comment:
Wouldn't `Return value` be more explicit?

Reply:
You are probably right, I didn't do it because the returned value is technically a builder struct, but I can add short sentence to clarify that in the section.